src/snapshot.rs

```diff
@@ -1,5 +1,19 @@
 // Copyright 2018 the Deno authors. All rights reserved. MIT license.
 use libdeno::deno_buf;
-extern "C" {
-  pub static deno_snapshot: deno_buf;
+use std;
+
+pub fn deno_snapshot() -> deno_buf {
+  let data =
+    include_bytes!(concat!(env!("GN_OUT_DIR"), "/gen/snapshot_deno.bin"));
+  let ptr = data.as_ptr();
+  // TODO The transmute is not necessary here. deno_buf specifies mutable
+  // pointers when it doesn't necessarally need mutable. So maybe the deno_buf
+  // type should be broken into a mutable and non-mutable version?
+  let ptr_mut = unsafe { std::mem::transmute::<*const u8, *mut u8>(ptr) };
```

Comment:
Do we need to use `deno_buf` to write memory? I think use `*const u8` in `deno_buf` is appropriate.
Nevertheless, ` ptr as *mut u8 ` is still simpler.